services/galley/test/integration/API/MLS.hs

```diff
@@ -215,6 +217,11 @@ tests s =
           test s "listing conversations without MLS configured" testSelfConversationMLSNotConfigured,
           test s "attempt to add another user to a conversation fails" testSelfConversationOtherUser,
           test s "attempt to leave fails" testSelfConversationLeave
+        ],
+      testGroup
+        "Typing indicators"
+        [ test s "POST /federation/on-typing-indicator-updated : Update typing indicator by remote user" updateTypingIndicatorFromRemoteUser,
+          test s "POST /federation/on-typing-indicator-updated : Update typing indicator to remote user" updateTypingIndicatorToRemoteUser
```

Comment:
These should be moved to `API`, I think. You don't need to use MLS in these tests. Also, there is a missing case: remote conversation with a local user sending a typing notification.